### PR TITLE
clean up folders and structure of project

### PR DESCRIPTION
**Pull request by Maxcutex**
**What does this PR do?**

Create folder structure for project

**Description of Task to be completed?**

Create folder structure for project

**Any background context you want to provide?**

None

**How should this be manually tested?**

Does not apply

**What are the relevant pivotal tracker stories?**

#158943992

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -12,6 +16,12 @@ class ApplicationController < Sinatra::Base
...
   end
-end 
\ No newline at end of file
+
+  get "/" do
+    "Hello world"
+  end 
+  
+end
+
```
Comment (igbanam) [inline app/controllers/application_controller.rb]: This extra line is not necessary

File: config.ru [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+#   if (pending = migrator.pending_migrations).any?
+#     raise %Q[Attention! The following migrations are PENDING:#{pending.map(&:filename).join("\n")}Please run 'rake db:migrate db:test:prepare'...]
+#   end
+# end
 use Rack::MethodOverride
-use UserController
-use CourseController
-run ApplicationController
\ No newline at end of file
+# suse UserController
```
Comment (igbanam) [inline config.ru]: typo?

File: config/environment.rb [hunk 3]
```diff
@@ -1,12 +1,31 @@
...
+  # set :show_exceptions, true
+  ActiveRecord::Base.establish_connection(
+    :adapter => "sqlite3",
+    :show_exceptions => true,
+    :database => "db/#{ENV['SINATRA_ENV']}.sqlite"
+  )
+end
+ 
+configure :production do
+  db = URI.parse(ENV['DATABASE_URL'] || 'postgres:///localhost/mydb')
```
Comment (igbanam) [inline config/environment.rb]: does this work with `postgres://localhost`?

Comment (Maxcutex) [reply config/environment.rb]: ve not created the db url yet. since its production. only test which is sqllite works

File: config.ru (hunk 2 above)
Comment (Maxcutex) [reply config.ru]: yes typo

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (Maxcutex) [reply app/controllers/application_controller.rb]: Thought extra line needs to be there

File: config/environment.rb (hunk 3 above)
Comment (igbanam) [reply config/environment.rb]: You have a development configuration

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (igbanam) [reply app/controllers/application_controller.rb]: Not necessarily

File: Gemfile [hunk 4]
```diff
@@ -1,13 +1,18 @@
...
 gem 'rack-test'
 gem 'shotgun'
 gem 'haml'
-gem 'rubocop'
\ No newline at end of file
+gem 'rubocop'
+gem 'pg'
+gem 'activerecord'
+gem 'sinatra-activerecord'
+gem 'sqlite3'
```
Comment (chadwalt) [inline Gemfile]: Add a blank line at the end of the file

File: Rakefile [hunk 5]
```diff
@@ -1,6 +1,8 @@
-require "bundler/gem_tasks"
-require "rspec/core/rake_task"
+require "sinatra/activerecord/rake"
+require './config/environment'
 
-RSpec::Core::RakeTask.new(:spec)
-
-task :default => :spec
+# namespace :db do
```
Comment (chadwalt) [inline Rakefile]: Remove the commented out code

File: config.ru [hunk 6]
```diff
@@ -1,10 +1,18 @@
 require './config/environment'
 
-if ActiveRecord::Migrator.needs_migration?
-  raise 'Migrations are pending. Run `rake db:migrate` to resolve the issue.'
-end
-
+# if ActiveRecord::Migrator.needs_migration?
```
Comment (chadwalt) [inline config.ru]: Remove commented out code

File: config/environment.rb [hunk 7]
```diff
@@ -1,12 +1,31 @@
...
 require 'rack-flash'
-Bundler.require(:default, ENV['SINATRA_ENV'])
+Bundler.require()
 
-ActiveRecord::Base.establish_connection(
-  :adapter => "sqlite3",
-  :database => "db/#{ENV['SINATRA_ENV']}.sqlite"
-)
+configure :development do
+  # set :database, 'sqlite:///dev.db'
```
Comment (chadwalt) [inline config/environment.rb]: Remove commented out code

File: config/environment.rb [hunk 8]
```diff
@@ -1,12 +1,31 @@
...
+    :adapter  => db.scheme == 'postgres' ? 'postgresql' : db.scheme,
+    :host     => db.host,
+    :username => db.user,
+    :password => db.password,
+    :database => db.path[1..-1],
+    :encoding => 'utf8'
+  )
+end
 
 require_all 'app'
```
Comment (chadwalt) [inline config/environment.rb]: Add a blank line at the end of the file.

**Pull request merged**